### PR TITLE
Annie/new data

### PR DESCRIPTION
**Pull request by anneCarlson**
Basic data upload script. @madisonmay @sihrc

### CODE REVIEW

File: cluster/search/load_data.py [hunk 1]
```diff
@@ -32,71 +42,171 @@ def parse_obj_to_document(obj):
...
+        clean_documents = futures[future]
         with open(data_dir+filename, 'w') as data_dump:
             for document in clean_documents:
                 data_dump.write(json.dumps(document)+'\n')
 
-    print 'done - total time:', time.time() - t0
-    return True
-            
+        print "written"
+        # es.upload(clean_documents)
```
Comment (anneCarlson) [inline cluster/search/load_data.py]: Needs to be uncommented eventually, but I don't want to fuck with the database while I'm playing with this unfinished code.

File: cluster/search/load_data.py [hunk 2]
```diff
@@ -32,71 +42,171 @@ def parse_obj_to_document(obj):
...
-            new_doc["indico"]["title_places"] = new_doc["indico"]["places"]
-            new_doc["indico"]["title_organizations"] = new_doc["indico"]["organizations"]
-
-            new_doc["indico"]["people"] = indicoio.people(text)
-            new_doc["indico"]["places"] = indicoio.places(text)
-            new_doc["indico"]["organizations"] = indicoio.organizations(text)
-
-            new_documents.append(new_doc)
-            new_doc = {}
+            text_analysis = indicoio.analyze_text([doc.get("title") for doc in documents_sub_list], apis=[
```
Comment (sihrc) [inline cluster/search/load_data.py]: don't use multi. its not reliable and more susceptible to a timeout error

File: cluster/search/load_data.py [hunk 3]
```diff
@@ -1,24 +1,34 @@
...
 
 def parse_obj_to_document(obj):
-    text = obj.get("content")
-    title = obj.get("title")
-    link = obj.get("link")
-    tags = obj.get("symbols")
+    text = obj.get("description_text")
+    title = obj.get("name_post", obj.get("name_story"))
+    link = obj.get("url_post", obj.get("url_story"))
+    tags = [obj.get("name_categorie", obj.get("name_topics"))]
```
Comment (sihrc) [inline cluster/search/load_data.py:60]: category?

**Comment by sihrc** [comment]
i would strongly recommend not changing any of the logic besides adding more data... if its working then by all means. Have we tried running it yet?

**Pull request merged**